pmaweb/templates/index.html

```diff
@@ -2,142 +2,147 @@
-        <li>And much more...</li>
-    </ul>
+<div class="hero hero-secondary">
+  <div class="content">
+    <h2>
+      <a href="{% url 'downloads' %}#features">Features</a>
+    </h2>
+    <div class="box">
+      <div class="box-horz">
+        <div class="box-item">
+          <div class="box-item-title">
+            Intuitive Web Interface
+          </div>
+          <div class="box-item-image">
+            <svg>
+              <use xlink:href="#pma-web-interface"></use>
+            </svg>
+          </div>
+        </div>
+        <div class="box-item">
+          <div class="box-item-title">
+            Support for most MySQL features
+          </div>
+          <div class="box-item-image">
+            <svg>
+              <use xlink:href="#pma-server"></use>
+            </svg>
+          </div>
+        </div>
+        <div class="box-item">
+          <div class="box-item-title">
+            Import and Export data from and to multiple formats
+          </div>
+          <div class="box-item-image">
+            <svg>
+              <use xlink:href="#pma-data-extraction"></use>
+            </svg>
+          </div>
+        </div>        
+        <div class="box-item">
+          <div class="box-item-title">
+            Manage multiple servers
+          </div>
+          <div class="box-item-image">
+            <svg>
+              <use xlink:href="#pma-maintenance"></use>
+            </svg>
+          </div>
+        </div>
+        <div class="box-item">
+          <div class="box-item-title">
+            Translated in 72 languages
+          </div>
+          <div class="box-item-image">
+            <svg>
+              <use xlink:href="#pma-around-the-world"></use>
+            </svg>
+          </div>
+        </div>
+        <div class="box-item">
+          <div class="box-item-title">
+            Full database searching
+          </div>
+          <div class="box-item-image">
+            <svg>
+              <use xlink:href="#pma-search"></use>
+            </svg>
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>  
 </div>
 
-<div id="latest-news">
-    <h2 class="entry-title"><a href="{% url 'news' %}">Latest News</a></h2>
-    <ul>
+<div class="hero">
+  <div id="latest-news" class="content">
+    <h2>
+      <a href="{% url 'news' %}">News</a>
+    </h2>
+    <div class="news-list">
     {% for item in short_news %}
-    <li>
-        <a href="{{ item.get_absolute_url }}">{{ item.date|date:"Y-m-d" }}: {{item.title}}</a>
-    </li>
+      <a class="news-item" href="{{ item.get_absolute_url }}">
+          <div class="news-date">
+            <div class="news-day">
+              {{ item.date|date:"d" }}
+            </div>
+            <div class="news-month" title="{{ item.date|date:'mmmm' }}">
+              {{ item.date|date:"mm" }}
+            </div>
+            <div class="news-year">
+              <!-- TODO: Needs a if(item.date|date:'yyyy' == current_year) {display:none;} -->
```

Comment:
todo ?

Reply:
I left this as TODO because I don't know how to do this on Django (and that news block isn't working on my local machine).

The idea is to hide the year if it's the current one. I may leave it here so you can fix it later.